### PR TITLE
Refactor public facing pages with Tailwind

### PR DESCRIPTION
**Pull request by jfranciscosousa**
Keep on tidying up CSS, redesigning the main page with Tailwind.

Refactored the books list and the gatherings list. Still need to fix some issues with moderator only actions and such, which are not currently mobile-friendly.

Any missing layout changes are not actually missing. I'll just touch them later.

### CODE REVIEW

**Comment by jpcorreia99** [comment]
Actually really like the redesign of the gatherings. Regarding the part about the books, can't exactly pinpoint where it is happening on the code, but the book covers are not in the correct dimensions, see image below: 
image

**Comment by jpcorreia99** [comment]
Also, since you're redesigning it (and without trying to overwork you), if you could change the popups a bit so that they do not overlap so drastically with the gathering info, I would appreciate it 
image

**Comment by jfranciscosousa** [comment]
Thanks for the review, I'll fix the book stuff. About the popups, I'll have to think about that later. Don't want to do everything in one go

File: app/assets/images/back_arrow.svg [hunk 1]
```diff
@@ -1 +1,3 @@
-<svg width="10" height="14" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M9 1L2 7l7 6" stroke="#7485A9" stroke-width="1.5"/></svg>
\ No newline at end of file
+<svg width="10" height="14" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path d="M9 1L2 7l7 6" stroke="currentColor" stroke-width="1.5"/>
+</svg>
```
Comment (DavideSilva) [inline app/assets/images/back_arrow.svg:3]: Missing newline

File: app/views/books/_book_list.erb [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                method: :delete,
+                data: {confirm: "Are you sure you want to delete it?" },
+                class: "absolute bottom-4 right-6"
+            %>
+          </div>
+        <% end %>
+      </div>
+    </div>
+  <% end %>
+</div>
```
Comment (DavideSilva) [inline app/views/books/_book_list.erb:76]: Missing newline

**Review by DavideSilva — APPROVED** [review]
Check the comments and :shipit:

**Review by jpcorreia99 — APPROVED** [review]
Thank you for keeping the project alive (and improving it). Just fix the failing test and :shipit:

**Pull request merged**